app/vmalert/remotewrite/init.go

```diff
@@ -13,7 +13,9 @@ import (
 
 var (
 	addr = flag.String("remoteWrite.url", "", "Optional URL to VictoriaMetrics or vminsert where to persist alerts state "+
```

Comment:
I think we should add support for datasource.url as well. 
The purpose of SRV discovery is to update IP addresses without need to restart the process. 
vmselect (datasource.url) has the same chance of IP change as vminsert(remotewrite.url). So it makes perfect sense to support both.

notifier.url could also support SRV discovery.

Reply:
Added for `-datasource.url`, `-remoteWrite.url` and `-remoteRead.url`, since all of them can be the same VM instance.
Do you think we should add for notifier.url now?

Reply:
Let's merge this without notifier.url, as it already can be discovered via notifier config.